test/helpers/reset-database.js

```diff
@@ -6,12 +6,13 @@ const vetVisits = require('../mocks/schemes/vet-visits')
 const cs = require('../mocks/schemes/cs')
 const bps = require('../mocks/schemes/bps')
 const fdmr = require('../mocks/schemes/fdmr')
+const sfi23 = require('../mocks/schemes/sfi23')
 const manual = require('../mocks/schemes/manual')
 const { sfiHoldCategory, sfiPilotHoldCategory, manualLedgerHoldCategory, debtEnrichmentHoldCategory } = require('../mocks/holds/hold-category')
 
 const resetDatabase = async () => {
   await db.sequelize.truncate({ cascade: true })
-  await db.scheme.bulkCreate([sfi, sfip, lumpSums, vetVisits, cs, bps, fdmr, manual])
+  await db.scheme.bulkCreate([sfi, sfip, lumpSums, vetVisits, cs, bps, fdmr, sfi23, manual])
```

Comment:
I'd put this below manual just to match the order of the schemes

Reply:
order amended, thanks!